darkside-tests/tests/tests.rs

```diff
@@ -118,6 +51,70 @@ async fn simple_sync() {
     );
 }
 
+#[tokio::test]
+async fn spend_orchard_notes() {
```

Comment:
I'd definitely add documentation on top of the test function so that the intent of the test is known beforehand